pom.xml

```
@@ -23,8 +23,8 @@
   </description>
 
   <properties>
-    <jenkins.baseline>2.249</jenkins.baseline>
-    <jenkins.version>${jenkins.baseline}.1</jenkins.version>
+    <jenkins.baseline>2.263</jenkins.baseline>
+    <jenkins.version>${jenkins.baseline}.4</jenkins.version>
```

Is it really required to bump to .4? Typically it is sufficient to declare `<jenkins.baseline>2.263</jenkins.baseline>`.
Then Jenkins version will be automatically set to 2.263.1. If possible, .1 LTS releases should be used. 

```suggestion
    <jenkins.baseline>2.263</jenkins.baseline>
```

---

@simonsymhoven ping

Is 2.263.1 sufficient?

---

> @simonsymhoven ping
> 
> Is 2.263.1 sufficient?

Sorry! Yes it is. I will change that, one second.